src/address-db.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * This is a module that uses a database dump of contract addresses and their deployed
+ * bytecode in the format
+ *
+ * {"000028ece24bca7ecd9615bef14cbf3720b7f1b87d095680bae61b6c6db5feb3":"0x8b88a39383c90000bf8be57b8abfd59a6256b2e2"}
+ * {"000034301b5c8f4049d759092cd11904f589d73e6283bc2ea1e27d3eadb84a84":"0x7ace493e27423bf13406e2b3ebe96aace38ae032"}
+ *
+ * (the first field is the keccak256-hash of the bytecode, the second is the address).
+ *
+ * The file has to be sorted and the field widths have to match exactly (i.e. the bytecode hash cannot
+ * have a "0x" prefix but the address has to have it).
+ */
+
+// TODO make the file name configurable.
+
+import fs from 'fs';
+import util from 'util';
+import Web3 from 'web3';
+
+const log = console.log;
+const dbFile : string = '/db/addressDB.txt';
```

Comment:
Move this to environment?

Reply:
I've ended up just commenting out the address-db references in injector.ts because there's no "write" operation to the db anywhere. 

I think this can be deferred until the second iteration - we'll need to add tests, make sure the volume is mounted etc etc..

Reply:
Tracking this #58